crates/statetest-types/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "revm-statetest-types"
+description = "State test types for revm"
+version = "1.0.0"
+authors.workspace = true
+edition.workspace = true
+keywords.workspace = true
+license.workspace = true
+repository.workspace = true
+readme.workspace = true
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+
+[lints.rust]
+unreachable_pub = "warn"
+unused_must_use = "deny"
+rust_2018_idioms = "deny"
+
+[lints.rustdoc]
+all = "warn"
+
+[dependencies]
+# revm
+revm = { workspace = true }
```

would be good to include `std` and `serde` feature
```suggestion
revm = { workspace = true, features = ["std", "serde"] }
```